packages/components/psammead-image-placeholder/package.json

```diff
@@ -18,12 +18,15 @@
   "homepage": "https://github.com/bbc/psammead/blob/latest/packages/components/psammead-image-placeholder/README.md",
   "dependencies": {
     "@bbc/psammead-assets": "^0.1.2",
-    "@bbc/psammead-styles": "^0.1.3",
-    "styled-components": "^4.1.2"
+    "@bbc/psammead-styles": "^0.1.3"
   },
   "devDependencies": {
     "@bbc/psammead-test-helpers": "^0.3.0",
-    "react": "^16.6.3"
+    "react": "^16.6.3",
+    "styled-components": "^4.1.3"
+  },
+  "peerDependencies": {
+    "styled-components": "^4.1.3"
```

Comment:
Shouldn't react be a peerDependency too? As it's used in index.jsx

Reply:
yep